ops/scripts/pipeline/check-deploy-bicep.sh

```diff
@@ -3,7 +3,7 @@
 # Title:        check-deploy-bicep.sh
 # Description:  Helper script to check if bicep changes have been made and if bicep should be deployed
 
-set -euo pipefail # ensure job step fails in CI pipeline when error occurs
+set -xeuo pipefail # ensure job step fails in CI pipeline when error occurs
 
 app_rg=$1
 deploy_flag=$2 #workflow dispatch parameter enableBicepDeployment from GHA
```

Comment:
I don't think need to pass this into the script. OR we could short circuit the logic if true is passed in. But I really don't think we need to pass this value. A false doesn't do anything and if its true we don't really need to run the script.